scripts/consume_sitemaps.py

```diff
@@ -170,62 +171,52 @@ def fetch_sitemap_files(authority, last_updated_date, thing_ids: typing.Dict[str
         sitemap_file_fetcher.fetch_sitemap_file()
         sitemap_file_iterator = sitemap_file_fetcher.url_iterator()
         thing_futures = []
+        # max_workers is 1 so that we may execute one request at a time in order to avoid overwhelming the server
+        # We still gain benefits as the next things request executes while the local processing is occurring
         with concurrent.futures.ThreadPoolExecutor(
-            max_workers=CONCURRENT_DOWNLOADS
+            max_workers=1
```

Comment:
I didn't fully understand this bit previously -- this limits the in-flight requests to 1 so we don't overwhelm the server.